src/plugin.hpp

```diff
@@ -229,21 +229,6 @@ struct JamesClockTracker {
 
 struct SequelClockTracker {
 
-    SequelClockTracker(
```

Comment:
I moved the loop in the constructor to the initailzeRows() method because:

1. I can then call it from the child class, which I think makes more sense
2. It's more clear what it's doing, compared to a random loop in the constructor